tools/grammar_nanny.nim

```diff
@@ -4,7 +4,7 @@
 import std / [strutils, sets]
 
 import ".." / compiler / [
-  llstream, ast, lexer, options, msgs, idents,
+  llstream, lexer, options, msgs, idents,
```

Comment:
Why not patch grammar_nanny to automate this?

Reply:
done; it reports these:
```
Unused non-terminal: enum
Unused non-terminal: inlTupleDecl
Unused non-terminal: object
Unused non-terminal: procExpr
Unused non-terminal: exceptBlock
Unused non-terminal: tupleClass
Unused non-terminal: distinct
Unused non-terminal: tupleConstr
Unused non-terminal: module
Unused non-terminal: extTupleDecl
```
which can be addressed in future PR

Reply:
Yes but now I need to look into why these are not used and what is used instead.